doc/widgets/univariatepolynomialregression.rst

```diff
@@ -40,40 +41,45 @@ Regressor coefficients if it has them.
 Description
 -----------
 
-This widget interactively shows regression line using different regressors from Orange3 Regression module provided on
-the input. In widget **Polynomial expansion** can be set. Polynomial expansion is regulation of the degree
-of polynom that is used to transform input data and has an effect on the shape of a curve. If polynomial expansion is
-set to 1 it means that untransformed data are used in regression.
+This widget interactively shows regression line using any of the regressors from *Orange3 Regression* module. 
+In the widget, `polynomial expansion <https://en.wikipedia.org/wiki/Polynomial_expansion>`__ can be set. 
+Polynomial expansion is a regulation of the degree of polynom that is used to transform the input data and has an effect 
+on the shape of a curve. If polynomial expansion is set to 1 it means that untransformed data are used in the regression.
 
-.. figure:: images/polyregressionmain.png
+.. figure:: images/polynomial-regression-stamped.png
 
-1. Provide data with at least two continuous attributes on input. Also provide any regression learner on input (if
-   not provided *Linear Regression* is used.
+1. Regressor name. Input data needs least two continuous attributes.
 
-3. Open widget and select **Imput** and **Target** feature in a dropdown menu.
+2. *Input*: independent variable on axis x.
 
-2. Select the degree of a polynomial expansion. With changing it you can see the difference between the
-   regression lines with different degree.
+   *Polynomial expansion*: degree of polynomial expansion.
 
-3. **Apply** button chan be used to submit changes on output.
+   *Target*: dependent variable on axis y.
+
+3. *Save Image* saves the image to the computer in a .svg or .png
+   format.
+   
+   *Report* includes widget parameters and visualization in the report.
 
 Example
 -------
 
-With *File* widget we loaded *Iris* data set. We also connect *Linear Regression* learner as show on figure above.
-In widget we selected *petal length* as an **Input** feature and *petal width* as a **Target**.
-We set **Polynomial expansion** to 1 what gives us linear regression line. Result is show on the figure below.
+.. figure:: images/polyregressionmain.png
+
+We loaded *iris* data set with the :doc:`File<../orange3/doc/visual-programming/source/widgets/data/file>` widget. 
+Then we connected :doc:`Linear Regression<../orange3/doc/visual-programming/source/widgets/regression/linear>` learner to the **File** widget.
```

Comment:
Here should be

    Then we connected :doc:`Linear Regression<../orange3/doc/visual-   programming/source/widgets/regression/linear>` learner to the **Polynomial Regression** widget.